CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## master / unreleased
+* [BUGFIX] Query Frontend: queries with negative offset should check whether it is cacheable or not. #5719
```

Comment:
Let's move this entry to the end to the end. It should be grouped together with `BUGFIX` items